### PR TITLE
Fix startup problems in benchmark (fixes #1329)

### LINKED ISSUE #1329: Failed to run benchmark tools
**Environment**: [Workbench](https://github.com/rubykube/workbench)

**Command**:
```sh
$ rake benchmark:execution
```
Peatio version: `1.8.39`

I got **error**:

```
NoMethodError: undefined method `update!' for nil:NilClass
/home/app/lib/benchmark/helpers.rb:15:in `block in lock_funds'
/home/app/lib/benchmark/helpers.rb:14:in `each'
/home/app/lib/benchmark/helpers.rb:14:in `lock_funds'
/home/app/lib/benchmark/matching.rb:26:in `block (3 levels) in run_prepare_orders'
/home/app/lib/benchmark/matching.rb:26:in `block (2 levels) in run_prepare_orders'
/home/app/lib/benchmark/matching.rb:24:in `block in run_prepare_orders'
/opt/vendor/bundle/ruby/2.5.0/gems/activesupport-4.2.10/lib/active_support/core_ext/range/each.rb:7:in `each'
/opt/vendor/bundle/ruby/2.5.0/gems/activesupport-4.2.10/lib/active_support/core_ext/range/each.rb:7:in `each_with_time_with_zone'
/home/app/lib/benchmark/matching.rb:22:in `map'
/home/app/lib/benchmark/matching.rb:22:in `run_prepare_orders'
/home/app/lib/benchmark/matching.rb:15:in `run'
/home/app/lib/tasks/benchmark.rake:18:in `block (2 levels) in <top (required)>'
/opt/vendor/bundle/ruby/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => benchmark:execution
(See full trace by running task with --trace)
```

Same issue with `rake benchmark:matching` command.

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Fixed benchmark tools #1329

### CODE REVIEW

File: app/trading/matching/executor.rb [hunk 1]
```diff
@@ -82,8 +82,14 @@ def publish_trade
     end
 
     def raise_error code, msg
-      raise TradeExecutionError.new \
-        ask: @ask.attributes, bid: @bid.attributes, price: @price, volume: @volume, funds: @funds, error: {code: code , message: msg}
+      raise TradeExecutionError.new(
```
Comment (yivo) [inline app/trading/matching/executor.rb]: Could you please rollback this? It intercepts with https://github.com/rubykube/peatio/pull/1335

File: app/trading/matching/constants.rb [hunk 2]
```diff
@@ -9,10 +9,11 @@ class InvalidOrderError   < StandardError; end
   class NotEnoughVolume     < StandardError; end
   class ExceedSumLimit      < StandardError; end
   class TradeExecutionError < StandardError
-    attr_accessor :options
+    attr_reader :options
```
Comment (yivo) [inline app/trading/matching/constants.rb]: Could you please rollback this? It intercepts with https://github.com/rubykube/peatio/pull/1335

**Review by yivo — CHANGES_REQUESTED** [review]

File: app/trading/matching/executor.rb (hunk 1 above)
Comment (dinesh-skyach) [reply app/trading/matching/executor.rb]: @yivo reverted

File: app/trading/matching/constants.rb (hunk 2 above)
Comment (dinesh-skyach) [reply app/trading/matching/constants.rb]: @yivo done

**Pull request merged**